### PR TITLE
Link loop/for keyword

### PR DESCRIPTION
**Pull request by pickfire**
Even though the reference already have all of these, I am just adding related keywords in the see also to let others easily click on the related keyword.

### CODE REVIEW

File: library/std/src/keyword_docs.rs [hunk 1]
```diff
@@ -879,6 +882,9 @@ mod while_keyword {}
 ///
 /// For more information on `loop` and loops in general, see the [Reference].
 ///
+/// See also, [`for`].
+///
+/// [`for`]: keyword.loop.html
```
Comment (camelid) [inline library/std/src/keyword_docs.rs:887]:
```suggestion
/// [`for`]: keyword.for.html
```

Comment (pickfire) [reply library/std/src/keyword_docs.rs]: Thanks a lot, I missed this.

**Comment by withoutboats** [comment]
Why not link while as well?

**Comment by pickfire** [comment]
> Why not link while as well?

Sorry, I forgot about while.

**Pull request merged**